src/main/java/org/prebid/server/bidder/interactiveoffers/InteractiveOffersBidder.java

```diff
@@ -38,9 +39,22 @@ public InteractiveOffersBidder(String endpointUrl, JacksonMapper mapper) {
 
     @Override
     public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+
+        ExtImpInteractiveoffers extImpInteractiveoffers = null;
+        String url = endpointUrl;
+        String partnerId;
+        try {
+            partnerId = request.getImp().get(0).getExt().get("bidder").get("partnerId") + "";
+            if (partnerId.length() > 2) {
+                partnerId = partnerId.substring(1, partnerId.length() - 1);
+            }
+            url += partnerId;
+        } catch (PreBidException e) {
+            return Result.withError(BidderError.badInput(e.getMessage()));
+        }
         return Result.withValue(HttpRequest.<BidRequest>builder()
```

Comment:
move this inside your try block

Reply:
Not sure what you'r asking to move inside the try block

Reply:
move this 
`        return Result.withValue(HttpRequest.<BidRequest>builder()
                .method(HttpMethod.POST)
                .uri(url)
                .headers(HttpUtil.headers())
                .payload(request)
                .body(mapper.encode(request))
                .build());`
to `try {` block 
and you dont need to create `        String url = endpointUrl;`